Tutorial.md

```diff
@@ -0,0 +1,66 @@
+# MSON Tutorial
+
+MSON is short for Markdown Syntax for Object Notation, and it's a way to represent data structures in a human readable plain text form. This tutorial will take you though the basics of MSON and how you can use MSON to describe a data structure.
+
+> **NOTE:** **Additional MSON Resources**
+>
+> + [MSON Specification](https://github.com/apiaryio/mson/blob/master/MSON%20Specification.md)
+> + [Examples](https://github.com/apiaryio/mson#notational-conventions)
+
+## Data structures in MSON
+
+To get started, we're going to create a data structure in MSON to represent a Person.
+
+A data structure to store a Person could be represented in MSON using the following:
+
+```apib
+# Person (object)
+
++ name
+```
+
+Here we've declared our own named type called "Person". This is indicated by using a heading. This is one or more `#` symbols followed by a string representing the name of our data structure, after the name we have declared in round brackets the type of data structure that a person is, in this case, it is an object.
+
+An object is constructed with properties, and in this case we've listed them below the heading in a list. Lists are created by preceding list items with either a `+`, `*` or `-`.
```

Comment:
> Lists are created by preceding list items with either a `+`, `*` or `-`.

Lists are created by putting each list item on its own line and preceding each line with either a `+`, `*` or `-`.